app/models/user.rb

```diff
@@ -414,4 +416,8 @@ def validate_username_length
         maximum: User.username_max_length)
       validator.validate(self)
     end
+
+    def not_email_consul_dev?
```

Comment:
This method doesn't appear to behave as its name suggests. Can we make a positive assertion and change the guard clause to an `unless`?
```suggestion
    def email_consul_dev?
```